mne/viz/_3d.py

```diff
@@ -353,12 +353,56 @@ def plot_trans(info, trans_fname='auto', subject=None, subjects_dir=None,
     return fig
 
 
+def _percent_to_control_points(limits, stc_data, colormap):
+    """Private helper function to convert percentiles to control points.
+
+    Note: If using 'mne_analyze', generate cmap control points for a directly
+    mirrored cmap for simplicity. I.e., no normalization to account for a
```

Comment:
@mainakjas, it's technically correct since I'm starting a sentence with the phrase. Thanks for bringing this up though. I'll change it since it's somewhat conspicuous and doc strings are not a place to be using technical grammar.